modeling-cmds/src/def_enum.rs

```diff
@@ -695,6 +695,20 @@ define_modeling_cmd_enum! {
             pub color: Color,
         }
 
+        /// Set the color of the tool lines for the scene.
+        #[derive(Debug, Clone, Serialize, Deserialize, JsonSchema, ExecutionPlanFromMemory, ModelingCmdVariantEmpty)]
+        pub struct SetCurrentToolColor {
```

Comment:
WDYT about making this `SetCurrentToolProperties` and having only one property, `color`? That way in the future if we want to customize the lines more (e.g. bolder lines) we can just add more to this struct.

(if we do this, we should make it `color: Option<Color>`, so that users can choose which properties to update and which to keep as-is)

Same for SetDefaultSystemColor, could be SetDefaultSystemProperties

Reply:
Great idea!